packages/service-library/src/servicelib/long_running_tasks/task.py

```diff
@@ -397,33 +448,30 @@ async def remove_task(
                 raise
```

Comment:
here for simplicitiy I would not take care of whether reraising or not. can't the one calling this take care of this?
I would suggest to always raise here, and whatever the client of this code takes care whether it want to tackle the exception or not